server/views/can-continue.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'form-layout.html' %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+
+{% block formContent %}
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l" id="pageTitle">{{ pageTitle }}</h1>
+
+        <p class="govuk-body">You’ll need to:</p>
+
+        <ul class="govuk-list govuk-list--bullet">
+          <li>describe the item and how it meets the exemption criteria</li>
+          <li>upload some photos of it</li>
+          <li>provide contact details</li>
+          {{ unorderedList | safe}}
```

Comment:
Appreciate what you are doing here but I'm not too keen on the way the HTML for the unordered list items appears in the  .JS file, it feels like a breakage of the 'separation of concerns'.

I think a better way to achieve what you want here is to pass in the 1 or 2 items as a string array, then use the HTML below to render the HTML element(s) based on an 'additionalSteps' string array:

```
<ul....>
      <li>describe the item and how it meets the exemption criteria</li>
      <li>upload some photos of it</li>
      <li>provide contact details</li>

      {% for additionalStep in additionalSteps %}
        <li id="additionalStep-{{ loop.index }}">{{ additionalStep }}</li>
      {% endfor %}
</ul>
```

you will then have IDs of additionalStep-1 and possibly aadditionalStep-2 depending on the items passed in the array (i.e. a 1-based indexing)

Reply:
Thanks Duncan. Can't say I was that keen on what I did myself. The way you've suggested looks like a better way of doing it so I'll get my code updated.